python/dalex/dalex/fairness/group_fairness/object.py

```diff
@@ -0,0 +1,160 @@
+from .checks import *
+from .plot import *
+from ..basics._base_objects import _FairnessObject
+from ..basics.checks import check_other_objects
+from ..._theme import get_default_config
+
+
+class GroupFairnessClassification(_FairnessObject):
+
+    def __init__(self, y, y_hat, protected, privileged, label, verbose=True, cutoff=0.5):
+
+        super().__init__(y, y_hat, protected, privileged, verbose)
+
+        cutoff = check_cutoff(self.protected, cutoff, verbose)
+        self.cutoff = cutoff
+
+        sub_confusion_matrix = SubgroupConfusionMatrix(y_true=self.y,
+                                                       y_pred=self.y_hat,
+                                                       protected=self.protected,
+                                                       cutoff=self.cutoff)
+
+        sub_confusion_matrix_metrics = SubgroupConfusionMatrixMetrics(sub_confusion_matrix)
+        df_ratios = calculate_ratio(sub_confusion_matrix_metrics, self.privileged)
+        parity_loss = calculate_parity_loss(sub_confusion_matrix_metrics, self.privileged)
+
+        self._subgroup_confusion_matrix_metrics_object = sub_confusion_matrix_metrics
+        self.subgroup_metrics = sub_confusion_matrix_metrics.to_horizontal_DataFrame()
+        self.parity_loss = parity_loss
+        self.result = df_ratios
+        self.label = label
+
+    def fairness_check(self, epsilon=0.8, verbose=True):
```

Comment:
Was wondering: `check`? `check_fairness`? usually a verb(+noun)

Reply:
Yeah I was about to name it that in R package but changed order to adhere to other function names. There was `fairness_heatmap`, `fairness_pca` etc. so it was nice to have something similar.